assets/ares.yaml

```diff
@@ -0,0 +1,25 @@
+---
+
+- type: application
```

Comment:
Is this an application or a model?

Reply:
According to the assets spreadsheet, it's an application, but the website gave little information about it. I'm leaning towards application based on what I can find about it though.